gamemode/sv_player.lua

```diff
@@ -203,81 +203,243 @@ function GM:PlayerDeathSound()
 	return true
 end
 
-function GM:PlayerSelectSpawn( pl )
-
-	local spawnPoints = {}
-
-
-	if pl:Team() == 3 then // props
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "info_player_terrorist" ) )
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "info_player_axis" ) )
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "info_player_combine" ) )
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "info_player_pirate" ) )
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "info_player_viking" ) )
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "diprip_start_team_blue" ) )
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "info_player_blue" ) )        
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "info_player_human" ) )
-	elseif pl:Team() == 2 then 
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "info_player_counterterrorist" ) )
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "info_player_allies" ) )
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "info_player_rebel" ) )
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "info_player_knight" ) )
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "diprip_start_team_red" ) )
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "info_player_red" ) )
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "info_player_zombie" ) )      
-	end
-
-	local Count = table.Count( spawnPoints )
-
-	if pl:Team() == 1 || Count == 0 then
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "info_player_start" ) )
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "gmod_player_start" ) ) -- (Old) GMod Maps
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "info_player_teamspawn" ) ) -- TF Maps
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "ins_spawnpoint" ) ) -- INS Maps
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "aoc_spawnpoint" ) ) -- AOC Maps
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "dys_spawn_point" ) ) -- Dystopia Maps
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "info_player_coop" ) ) -- SYN Maps
-		spawnPoints = table.Add( spawnPoints, ents.FindByClass( "info_player_deathmatch" ) )
-	end
-
-
-	// recount
-	local Count = table.Count( spawnPoints )
-	
-	if ( Count == 0 ) then
-		Msg("[PlayerSelectSpawn] Error! No spawn points!\n")
-		return nil
+
+-- This is only a shallow copy.
+local function MergeTables(...)
```

Comment:
This isn't properly followed in a couple of places, but if it's `local`, then I'd prefer it to be `camelCase` to better distinguish that it's defined in a specific file. Similarly for other functions / variables being added, if possible.

There are a few exceptions like `PlayerMeta` or `EntityMeta` which are named for classes, though.